src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -405,11 +411,14 @@ static int synapseExceptionStatus(SynapseServerException e) {
 		// and we don't expect others. For the rest we'll just return 500.
 		return 500;
 	}
-
+	
 	@Override
 	public void doGet(HttpServletRequest req, HttpServletResponse resp)
 			throws IOException {
 		try {
+			if (req.isSecure()) {
+				resp.setHeader(StrictTransportSecurityHeaderName, StrictTransportSecurityHeaderValue);
```

Comment:
`setHeader` is used here and subsequently used in `doGetIntern` method as well.  according to the [java docs](https://www.oreilly.com/library/view/head-first-servlets/9780596516680/ch04s30.html#:~:text=The%20difference%20depends%20on%20whether,in%20exactly%20the%20same%20way.)  the `setHeader` will replace the existing header value.  I assume we want to send multiple headers in some cases?